quickstarts/audit/infrastructure-integrations-data-analysis/config.yml

```diff
@@ -46,3 +46,14 @@ documentation:
 
 icon: logo.svg
 website: https://www.newrelic.com
+
+dashboards:
+  - aws-integrations-data-ingest-analysis
+dashboards:
+  - azure-integrations-data-ingest-analysis
+dashboards:
+  - gcp-integrations-data-ingest-analysis
+dashboards:
+  - kubernetes-data-ingest-analysis
+dashboards:
+  - on-host-integrations-data-analysis
```

Comment:
This doesn't seem right. Need to verify that the quickstart uses all these dashboards, then update it to one list